src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java

```diff
@@ -104,8 +104,7 @@ public BlockLoc getBlock(int x, int y, int z) {
         return new BlockWrapper(this, x, y, z);
     }
 
-    @Override
-    public Biome getBiome(Vector3d position) {
+    public BiomeType getBiome(Vector3d position) {
```

Comment:
No offense, but why would I do that? Why can't I update the submodules? (Again, no offense intended.)